### DISCUSSION TITLE
Организации. Совместная работа с группами.

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#79] [Опросы]
Предлагаю ознакомится перед принятием решения.
Организации. Совместная работа с группами.
https://docs.github.com/ru/organizations/collaborating-with-groups-in-organizations/about-organizations
У меня этот вопрос в голове возник, и я хочу им поделиться.
Можно сделать трансфер репозитория в созданную публичную организацию, и превратить из личного репозитория пользователя audetv в общественный. Мне нужна помощь в решении этого вопроса, конечно, это не тот вопрос, который надо решать срочно или без него не сможем работать. Но это возникло в моей психике и требует разрешения и оценки.
Я за то, что бы преобразовать.

**Comment by audetv** [comment]
И да, я еще хотел опробовать функционал опросов.

**Comment by audetv** [comment]
И да еще из этого выходят связанный вопрос, который надо решить - наименование организации, в том числе латиницей для гитхаба. 
В любом случае, я это огласил и вывел в общественное обсуждение. Теперь это не моя идея )

**Comment by iprst** [comment]
Я думаю рано или поздно это преобразование может понадобится, но кажется не прямо сейчас. В любм случае гитхаб это структура, как и любой другой сайт, поэтому про чисто бесструктурное управление при участии внутри структуры говорить особо не приходится. То есть не критично на данный момент держаться за это определение. Если структура формируется, значит это кому-нибудь нужно (с)
